projects/storefrontstyles/scss/components/cart/_cart-coupon.scss

```diff
@@ -17,4 +29,49 @@
       padding: 0;
     }
   }
+  .cx-apply-voucher {
+    position: relative;
+  }
+
+  .cx-customer-coupons {
+    display: none;
+    z-index: 100;
+    position: absolute;
+    max-height: 130px;
+    overflow: auto;
+    width: 100%;
+    color: #000;
+    background-color: #fff;
+    font-size: var(--cx-font-small, 1rem);
```

Comment:
Please this to improve it:
https://github.com/SAP/cloud-commerce-spartacus-storefront/pull/5292#discussion_r353534764